clients/web/package.json

```diff
@@ -9,6 +9,7 @@
     "@emotion/css": "11.13.5",
     "@emotion/react": "11.14.0",
     "@freestar/pubfig-adslot-react-component": "3.6.2",
+    "@mozilla-services/ads-sdk": "git+ssh://git@github.com:mozilla-services/ads-sdk.git",
```

Comment:
Should note this isn't working and I am currently just manually placing the ads-sdk `dist` folder into the `clients/web/node_modules/@mozilla-services/` folder for now just to boostrap. 

We should be able to try vending for real tomorrow 🤞

Reply:
Still no vending unfortunately, a helpful SRE is trying to figure out the right npm permissions so that I can add the package without errors.